src/frontend/js/hooks/useCourseOrders/index.ts

```diff
@@ -27,8 +28,23 @@ const props: UseResourcesProps<
   messages,
 };
 
-/**
- * Joanie Api hook to retrieve/update a contract owned by the authenticated user.
- */
-export const useCourseOrder = useResource(props);
-export const useCourseOrders = useResources(props);
+export const useCourseOrder = (
+  id: NestedCourseOrder['id'],
+  filters?: CourseOrderResourceQuery,
+  queryOptions?: QueryOptions<NestedCourseOrder>,
+) => {
+  if (filters?.organization_id === OPTION_VALUE_ALL) {
+    delete filters.organization_id;
+  }
+  return useResource(props)(id, filters, queryOptions);
+};
+
+export const useCourseOrders = (
+  filters?: CourseOrderResourceQuery,
+  queryOptions?: QueryOptions<NestedCourseOrder>,
+) => {
+  if (filters?.organization_id === OPTION_VALUE_ALL) {
+    delete filters.organization_id;
+  }
+  return useResources(props)(filters, queryOptions);
+};
```

Comment:
Not quit sure to understand why you need to do that here ?

From my point of view, if we want to get data from all organizations, the filter `organization_id` should be simply undefined.

Reply:
I like this ALL const value.
But it'll redo it with undefined.

Reply:
What do you think @NathanVss? 
Do you think we should keep a little complex logic to add a `"All"` option in the select or remove this logic and use the `clearable` option of the select to get resource without organization filter ?